superset-frontend/cypress-base/cypress/integration/explore/AdhocFilters.test.ts

```diff
@@ -33,14 +33,15 @@ describe('AdhocFilters', () => {
 
   let numScripts = 0;
 
-  it('Should load AceEditor scripts when needed', () => {
+  xit('Should load AceEditor scripts when needed', () => {
```

Comment:
@kgabryje I was seeing errors on these in dev, fyi.

Reply:
Can we do a follow up to re-enable these tests if it's not too much trouble?

Reply:
yeah, I'll circle back on them.